server/routes/searchRouter.ts

```diff
@@ -15,10 +16,15 @@ export default function searchRouter({ auditService, searchService }: Services):
   })
 
   get('/results', async (req, res, next) => {
+    // TODO: replace this with FormData object
+    const searchItem: Order = {
+      dataType: 'am',
+      legacySubjectId: 1,
+    }
     await auditService.logPageView(Page.SEARCH_RESULTS_PAGE, { who: res.locals.user.username, correlationId: req.id })
 
     try {
-      const orders = await searchService.getOrders()
+      const orders = await datastoreSearchService.getOrders(searchItem)
       const tabulatedOrders = tabluateOrders(orders)
       res.render('pages/searchResults', { data: tabulatedOrders })
     } catch {
```

Comment:
I think this should also be searchForOrders - and in fact, should be pushed down into the controller, with the tabulating logic done on the controller layer, and the view only parsing FormData.
```suggestion
try {
  const searchResults = await searchController.search(searchItem)
  res.render('pages/searchResults', { data: searchResults })
} catch { 
  ...
}
```

Reply:
I will be removing it when I get back implementing the searchForm